data/templates/ssh/sshd_config

```diff
@@ -15,10 +15,17 @@ HostKey {{ key }}{% endfor %}
 # https://infosec.mozilla.org/guidelines/openssh
 # ##############################################
 
-# Keys, ciphers and MACS
-KexAlgorithms curve25519-sha256@libssh.org,ecdh-sha2-nistp521,ecdh-sha2-nistp384,ecdh-sha2-nistp256,diffie-hellman-group-exchange-sha256
-Ciphers chacha20-poly1305@openssh.com,aes256-gcm@openssh.com,aes128-gcm@openssh.com,aes256-ctr,aes192-ctr,aes128-ctr
-MACs hmac-sha2-512-etm@openssh.com,hmac-sha2-256-etm@openssh.com,umac-128-etm@openssh.com,hmac-sha2-512,hmac-sha2-256,umac-128@openssh.com
+{%- if security_ciphers_compatibility == "intermediate" -%}
```

Comment:
Hm thinking about this I really think we should split this setting into "nginx" and "ssh" config ... I know we technical people like consistency, but consider the following : 
- a yunohost admin might have users with not-so-recent devices and wants to keep intermediate compatibility
- but wants to harden the SSH conf (only used for administration in his/her use-case)

Reply:
Ok. Shall I make 2 differents PR ?

Reply:
Nah I think that's the same "feature" :wink: It's just a matter of having two different settings instead of one. (But idk, that's a decision decision, maybe other people have different views on this..)

Reply:
Done.